samples/snippets/fsharp/azure/queue-storage.fsx

```diff
@@ -12,102 +11,88 @@ let storageConnString = "..." // fill this in from your storage account
 let storageConnString = 
     CloudConfigurationManager.GetSetting("StorageConnectionString")
 *)
-//
-// Parse the connection string.
-//
-
-// Parse the connection string and return a reference to the storage account.
-let storageAccount = CloudStorageAccount.Parse(storageConnString)
 
 //
 // Create the Queue Service client.
 //
 
-let queueClient = storageAccount.CreateCloudQueueClient()
-
-//
-// Create a queue.
-//
-
-// Retrieve a reference to a container.
-let queue = queueClient.GetQueueReference("myqueue")
-
-// Create the queue if it doesn't already exist
-queue.CreateIfNotExists()
+let queueClient = new QueueClient(storageConnString, "myqueue");
+queueClient.CreateIfNotExists()
 
 //
 // Insert a message into a queue.
 //
 
 // Create a message and add it to the queue.
-let message = new CloudQueueMessage("Hello, World")
-queue.AddMessage(message)
+let message = "Hello, World"
```

Comment:
Given that this is a primitive now, it may be better to collapse 27 and 28 into a single line.    The comment on L26 should also be adjusted.

Is this restricted to `string` or is there a more complex flow that we should also consider demonstrating?

Reply:
Thanks for your guidance! I have adjust it.

Parameter `message` in the method `AddMessage` is only support two types `string` and `BinaryData` now.